string object_name
---
geometry_msgs/Pose pre_grasp
geometry_msgs/Pose grasp
